### DISCUSSION TITLE
How to see only filtered items (compare started with) when viewing left/right unique items?

### DISCUSSION THREAD

**Discussion by funbotix** [WinMerge/winmerge#1742] [Q&A]
When I start WinMerge using a file filter (EX: *.txt) and pick two folders to compare, that works fine. However, if I then enable viewing left and/or right unique items I see all kinds of files that don't have the .txt extension. Is there some way to constrain the view of unique items to the same filter that was used to start the compare operation?

Thanks!

**Comment by sdottaka** [comment]
That sounds like a bug, but I have created folders like the ones in the attached image and tried to compare them, but could not get it to reproduce. Do you mean the display when the comparison is in progress, not when it is completed?

## file filter = *.*

![image](https://user-images.githubusercontent.com/98126/222854316-65a9adf2-4d94-42a3-bc78-198c3e182890.png)

## file filter = *.txt

![image](https://user-images.githubusercontent.com/98126/222854398-3766a2a8-519f-46a9-a83f-1efb84906f6c.png)
